src/articles/2020/cdk-tips-01-how-to-use-local-cdk-commands.md

```diff
@@ -176,7 +176,8 @@ which ships with all modern distributions of NodeJS.
 So, instead of invoking `cdk init --language=<lang>`
 you would call `npx cdk init --language`,
 and `npx` will download the latest version of the `aws-cdk` package,
-and store it in a temporary directory.
+and store it in a temporary directory. You can use a specific `aws-cdk` package version if needed:
```

Comment:
If you notice, the lines are broken on sentence boundaries in this file. This makes the diff for changes like you're trying to add here look much nicer 🙂.

```suggestion
and store it in a temporary directory.
You can use a specific `aws-cdk` package version if needed:
```